website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+    We also know that many people developing in dbt have a preference for local development, where they can use their favorite terminal, text editor, keybindings, color scheme, and so on. This includes people with data engineering backgrounds, as well as those analytics engineers who started writing code in the dbt Cloud IDE and have expanded their skills. 
+
+    The new dbt Cloud CLI offers the best of both worlds, including: 
+
+    - The power of developing against the dbt Cloud platform 
+    - The flexibility of your own local setup
+
+    Run whichever community-developed plugins, pre-commit hooks, or other arbitrary scripts you like.
+
+    Some of the unique capabilities of this dbt Cloud CLI include:
+
+    - Automatic deferral of build artifacts to your Cloud project's production environment
+    - Secure credential storage in the dbt Cloud platform
+    - Support for dbt Mesh ([cross-project `ref`](/docs/collaborate/govern/project-dependencies))
+    - Development workflow for dbt Semantic Layer
+    - Speedier, lower cost builds
+
+    Refer to [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) to learn more.
+
+  </expandable>
+
+- <expandable alt_header='Custom branch fix'>
+
+    If you don't set a [custom branch](/docs/dbt-cloud-environments#custom-branch-behavior) for your dbt Cloud environment, it now defaults to the default branch of your Git repository (for example, `main`). Previously, [CI jobs](/docs/deploy/ci-jobs) would run for pull requests (PRs) that were opened against _any branch_ or updated with new commits if the **Custom Branch** option wasn't set. 
+
+    ## Azure DevOps 
+
+    Your Git pull requests (PRs) might not trigger against your default branch if you're using Azure DevOps and the default branch isn't `main` or `master`. To resolve this, [set up a custom branch](/faqs/Environments/custom-branch-settings) with the branch you want to target.  
+
+  </expandable>
+
+- <expandable alt_header='dbt deps auto install'>
+
+    The dbt Cloud IDE and dbt Cloud CLI now automatically installs `dbt deps` when your environment starts or when necessary. Previously, it would prompt you to run `dbt deps` during initialization. 
+
+    This improved workflow is available to all multi-tenant dbt Cloud users (Single-tenant support coming next week) and applies to dbt versions.
+
+    However, you should still run the `dbt deps` command in these situations:
+
+    - When you make changes to the `packages.yml` or `dependencies.yml` file during a session
+    - When you update the package version in the `packages.yml` or `dependencies.yml` file. 
+    - If you edit the `dependencies.yml` file and the number of packages remains the same, run `dbt deps`. (Note that this is a known bug dbt Labs will fix in the future.)
+
+  </expandable>
+
```

Comment:
i think we're missing the dbt explorer public preview: https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-8635587fd7a2eb828a4db1e1019090d808d69213b5f4138f373b9185cb5a2510L1-L10

Reply:
oh nvm public preview isn't included as you said in the description - resolving this!